administrator/language/en-GB/com_users.ini

```
@@ -395,6 +395,8 @@ COM_USERS_USERS_N_ITEMS_DELETED="%d users deleted."
 COM_USERS_USERS_N_ITEMS_DELETED_1="User deleted."
 COM_USERS_USERS_TABLE_CAPTION="Users"
 COM_USERS_USER_ACCOUNT_DETAILS="Account Details"
+COM_USERS_USER_ACTIVATE_AND_MAIL="Activate & Send Email"
```

```suggestion
COM_USERS_USER_ACTIVATE_AND_MAIL="Activate &amp; Send Email"
```

See https://github.com/joomla/joomla-cms/blob/f3b89f128c24cf4205b0ba54c7133cc091b0f7d9/administrator/language/en-GB/joomla.ini#L122